### PR TITLE
change test case code org.apache.pulsar.io.PulsarFunctionE2ETest#testFunctionAutomaticSubCleanup

### PR DESCRIPTION
**Pull request by zplinuxlover**
in testcase org.apache.pulsar.io.PulsarFunctionE2ETest#testFunctionAutomaticSubCleanup,
1.  there is one test checkpoint, we should not check `admin.functions().getFunction(tenant, namespacePortion, functionName).getCleanupSubscription()`, 
we shold check `admin.functions().getFunction(tenant, namespacePortion, functionName)` is not null, 
and `admin.functions().getFunction(tenant, namespacePortion, functionName).getCleanupSubscription()` should not be null

2. we should not check `Parallelism`, because the code here is 1 instead of 2

### CODE REVIEW

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java [hunk 1]
```diff
@@ -2552,9 +2552,10 @@ protected void internalSetOffloadPolicies(AsyncResponse asyncResponse, OffloadPo
             final String path = path(POLICIES, namespaceName.toString());
             byte[] content = globalZk().getData(path, null, nodeStat);
             Policies policies = jsonMapper().readValue(content, Policies.class);
-
-            if (offloadPolicies.getManagedLedgerOffloadDeletionLagInMillis()
-                    .equals(OffloadPolicies.DEFAULT_OFFLOAD_DELETION_LAG_IN_MILLIS)) {
+            // there is possibility that `offloadPolicies.getManagedLedgerOffloadDeletionLagInMillis()` is null.
+            if (offloadPolicies.getManagedLedgerOffloadDeletionLagInMillis() == null && OffloadPolicies.DEFAULT_OFFLOAD_DELETION_LAG_IN_MILLIS == null
+                    || offloadPolicies.getManagedLedgerOffloadDeletionLagInMillis() != null
+                    && offloadPolicies.getManagedLedgerOffloadDeletionLagInMillis().equals(OffloadPolicies.DEFAULT_OFFLOAD_DELETION_LAG_IN_MILLIS)) {
```
Comment (codelipenghui) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java:2558]: Looks not related to this PR right?

Comment (zplinuxlover) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java:2558]: the change of code is pulsar-broker/src/test/java/org/apache/pulsar/io/PulsarFunctionE2ETest.java, 
`pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java` is another problem, why the code appear here?

Comment (zplinuxlover) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java:2558]: is another issue 6775

Comment (codelipenghui) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java:2558]: So we can close #6775 as well.

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java [hunk 2]
```diff
@@ -2595,7 +2595,7 @@ protected void internalSetOffloadPolicies(AsyncResponse asyncResponse, OffloadPo
             final String path = path(POLICIES, namespaceName.toString());
             byte[] content = globalZk().getData(path, null, nodeStat);
             Policies policies = jsonMapper().readValue(content, Policies.class);
-
+            // there is possibility that `offloadPolicies.getManagedLedgerOffloadDeletionLagInMillis()` is null.
```
Comment (codelipenghui) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java:2598]: Shall we need to add this comment? look we already have a null check for deletion lag.

File: pulsar-broker/src/test/java/org/apache/pulsar/io/PulsarFunctionE2ETest.java [hunk 3]
```diff
@@ -1509,8 +1513,9 @@ public void testFunctionAutomaticSubCleanup() throws Exception {
 
         retryStrategically((test) -> {
             try {
+                // should remove `result.getParallelism() == 2` because the parallelism here should 1
                 FunctionConfig result = admin.functions().getFunction(tenant, namespacePortion, functionName);
-                return result.getParallelism() == 2 && result.getCleanupSubscription() == false;
+                return /*result.getParallelism() == 2 &&*/ result.getCleanupSubscription() == false;
```
Comment (codelipenghui) [inline pulsar-broker/src/test/java/org/apache/pulsar/io/PulsarFunctionE2ETest.java:1518]: You can fix the wrong assert directly.

**Comment by jiazhai** [comment]
Hi @zplinuxlover, Would you please help handle @codelipenghui 's comments?

File: pulsar-broker/src/test/java/org/apache/pulsar/io/PulsarFunctionE2ETest.java (hunk 3 above)
Comment (zplinuxlover) [reply pulsar-broker/src/test/java/org/apache/pulsar/io/PulsarFunctionE2ETest.java:1518]: OK

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java (hunk 2 above)
Comment (zplinuxlover) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java:2598]: OK

**Comment by jiazhai** [comment]
Since this is more related with tests, would like to remove it from 2.5.2, if it is really needed, we can add it back.
